### ISSUE TITLE
"Can not start node.js process" from Atom on fresh Windows install

### ISSUE THREAD

**Issue by joelhoward0** [wallabyjs/public#384]
I recently paved my machine (Windows 8, 64 bit) and reinstalled Node, Atom, and Wallaby.

Node seems to be working fine in general - it's on my PATH and running 'node -v' reports the correct result (4.2.3), but I see an error message when I try to start Wallaby:

"Can not start node.js process (node), make sure your system has node.js installed"

When I set the node path explicitly in my atom script (to the correct path, confirmed from the command line) I get:

"Can not start node.js process (node), make sure your system has node.js installed"

I'm at my wit's end. Is there a start log somewhere, or more info I can provide to help resolve this?

**Comment by ArtemGovorov** [comment]
Did you restart Atom after installing node?

> When I set the node path explicitly in my atom script

It doesn't look like the path you set is used by wallaby, otherwise the message would look like:
`Can not start node.js process (your/nodejs/path), make sure your system has node.js installed`.

Please make sure you set the node path for wallaby in your Atom init script correctly:
``` coffeescript
process.env.WALLABY_NODE = 'your/nodejs/path'
```
and restart Atom after the change.

**Comment by joelhoward0** [comment]
Thanks for taking a look at this. I got Wallaby to use the path I had set and got the message you described, but it didn't fix my problem even after restarting Atom.

I had also tried uninstalling and reinstalling Node, Wallaby, and Atom in various sequences, but nothing worked.

Eventually I deleted my Atom config/plugins folder (which was at C:\Users\MyUserName\.atom) and then reinstalling fixed this issue. I assume something to corrupted at install time, and wasn't being deleted during uninstall of either Atom or Wallaby.

Issue closed - back to blissful javascript coding. Thanks!
